Scripts/restoreazuredb.ps1

```diff
@@ -0,0 +1,49 @@
+param (
+    [parameter(Mandatory=$true)]
+    [string]$dbname,
```

Comment:
I suspect all parameters are derivable. See previous comments.

Reply:
Hi Rob, The parameter value $dbname will be coming as a secret value from pipeline